config/packages/prod/deprecations.yaml

```diff
@@ -0,0 +1,8 @@
+# As of Symfony 5.1, deprecations are logged in the dedicated "deprecation" channel when it exists
+#monolog:
+#    channels: [deprecation]
+#    handlers:
+#        deprecation:
+#            type: stream
+#            channels: [deprecation]
+#            path: php://stderr
```

Comment:
Why should not output any deprecations messages in production.

Reply:
Flex defaults